tests/e2e/TestConstants.ts

```diff
@@ -223,6 +223,11 @@ export const TestConstants = {
     /**
      * Print all timeout variables when tests launch, defaulte to false
      */
-    TS_SELENIUM_PRINT_TIMEOUT_VARIABLES: process.env.TS_SELENIUM_PRINT_TIMEOUT_VARIABLES || false
+    TS_SELENIUM_PRINT_TIMEOUT_VARIABLES: process.env.TS_SELENIUM_PRINT_TIMEOUT_VARIABLES || false,
+
+    /**
+     * URL of the workspace created by devworkspace-controller
+     */
+    TS_SELENIUM_DEVWORKSPACE_URL: process.env.TS_SELENIUM_DEVWORKSPACE_URL
```

Comment:
it's actually URL to devfile which is used to create devworkspace.

IMHO, TS_SELENIUM_DEVWORKSPACE_RAW_DEVFILE_URL is more relevant to value.

Reply:
It url of existed workspace.

Reply:
I see. We can keep it as proof of conception, but in future there should be factory URL with link to raw devfile, as far as I know.